src/main/resources/log4j2.xml

```diff
@@ -8,7 +8,7 @@
         </Lambda>
     </Appenders>
     <Loggers>
-        <Root level="info">
+        <Root level="debug">
```

Comment:
Are you intending to keep the logging at debug level?

Reply:
Yep!